revised charter for discussion (#8716)
hyperledger/besu: CHARTER.md

2. **Maintainer Oversight**

      1. The maintainers of the project (collectively, the "Maintainers")  will be responsible for all technical oversight of the open source Project.

      2. At the inception of the project and going forward, the Maintainers of the Project will be as set forth within the "MAINTAINERS" file within the Project's code repository. The Maintainers may choose an alternative approach for determining the voting members of the Project (such as the establishment of a technical steering committee for project technical oversight), and any such alternative approach will be documented in the MAINTAINERS file.  Any meetings of the Maintainers are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.

      3. Sub-projects generally will involve Contributors and Maintainers. The Maintainers  may adopt or modify roles so long as the roles are documented in the MAINTAINERS file. Unless otherwise documented:

         1. Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project;

>          2. Maintainers are Contributors who have earned the ability to modify ("commit") source code, documentation or other technical artifacts in a project's repository; and  

Reviewer: documentation repo has separate maintainers so maybe that needs to be discussed separately

>          3. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. A Maintainer may be removed by a majority approval of the other existing Maintainers.

Reviewer 2: PR https://github.com/hyperledger/besu/pull/8931 raised to ensure the rule on voting out a maintainer isn't lost.

>          3. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. A Maintainer may be removed by a majority approval of the other existing Maintainers.

Reviewer: think this is resolved now with #8937 having merged

>          3. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. A Maintainer may be removed by a majority approval of the other existing Maintainers.

Reviewer 3: Yeah agree this doesn't follow current process. It should just delegate to the process defined in the `MAINTAINERS.md` file

>          3. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. A Maintainer may be removed by a majority approval of the other existing Maintainers.

Reviewer: applied suggestion

         1. coordinating the technical direction of the Project;

         2. approving project or system proposals (including, but not limited to, archiving and changing a sub-project's scope);

         3. organizing sub-projects and removing sub-projects;

         4. creating sub-committees or working groups to focus on cross-project technical issues and requirements;

         5. appointing representatives to work with other open source or open standards communities;

         6. establishing community norms, workflows, issuing releases, and security issue reporting policies;

         7. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the "Series Manager") to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;

         8. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and

         9. coordinating communications regarding the Project.

3. **Voting**

>    1. While the Project aims to operate as a consensus-based community, if any  decision of the Maintainers requires a vote to move the Project forward, the voting members of the Maintainers will vote on a one vote per voting member basis.

Reviewer: we don't have voting members? where would this be defined

Reviewer: PR to clarify part of the voting process #8937

Reviewer: resolved with #8937 having merged

>    1. While the Project aims to operate as a consensus-based community, if any  decision of the Maintainers requires a vote to move the Project forward, the voting members of the Maintainers will vote on a one vote per voting member basis.

Reviewer: this now matches the gdoc

   2. It is anticipated that most decisions will be made asynchronously, but meetings of Maintainers may be held.  Quorum for such meetings requires at least fifty percent of all Maintainers to be present. Meetings of Maintainers may continue if quorum is not met but will be prevented from making any decisions at the meeting.

   3. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made asynchronously by electronic vote without a meeting require a majority vote of the Maintainers.

>    4. In the event a vote cannot be resolved by the Maintainers, any Maintainer may refer the matter to the Series Manager for assistance in reaching a resolution.

Reviewer: asked this question in the google doc

      5. The Project may seek to integrate and contribute back to other open source projects ("Upstream Projects"). In such cases, the Project will conform to all license requirements of the Upstream Projects, including dependencies, leveraged by the Project.  Upstream Project code contributions not stored within the Project's main code repository will comply with the contribution process and license terms for the applicable Upstream Project.

   3. The Maintainers may approve the use of an alternative license or licenses for inbound or outbound contributions on an exception basis. To request an exception, please describe the contribution, the alternative open source license(s), and the justification for using an alternative open source license for the Project. License exceptions must be approved by a two-thirds vote of all of the Maintainers.

   4. Contributed files should contain license information, such as SPDX short form identifiers, indicating the open source license or licenses pertaining to the file.

8. Amendments

>    1. This charter may be amended by a two-thirds vote of all of the Maintainers and is subject to approval by LF Projects.

Reviewer: ```suggestion
   1. This charter may be amended by an absolute majority vote of Maintainers and is subject to approval by LF Projects.
```

>    1. This charter may be amended by a two-thirds vote of all of the Maintainers and is subject to approval by LF Projects.

Reviewer: agree. made a suggestion
